src/web/page/handlebars.rs

```diff
@@ -16,7 +16,14 @@ lazy_static::lazy_static! {
 }
 
 fn load_rustc_resource_suffix() -> Result<String, failure::Error> {
-    let conn = crate::db::connect_db()?;
+    // New instances of the configuration or the connection pool shouldn't be created inside the
+    // application, but we're removing handlebars so this is not going to be a problem in the long
+    // term. To avoid wasting resources, the pool is hardcoded to only keep one connection alive.
```

Comment:
Why not pass in the connection as a parameter?

Reply:
It's not an easy change, since that function is called from the `lazy_static!`. I'd need to make something like #841, but I don't want to bother with it since @Kixiron will hopefully remove handlebars soon (:heart:).

Reply:
Got it, I agree that the change isn't worth the effort then.